support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.jsx

```diff
@@ -48,10 +49,12 @@ const content = (
     >
       <CampaignHeader />
       {paperHasDeliveryEnabled() &&
-        <Content needsHigherZindex>
+        <Content needsHigherZindex innerBackground="grey">
           <Text>
             <LargeParagraph>
-              We offer two different subscription types: voucher booklets and home delivery.
+              {useDigitalVoucher === null || useDigitalVoucher === false || useDigitalVoucher === undefined
```

Comment:
can we tighten up the type of useDigitalVoucher so it is guaranteed to be false or true to avoid this kind of belt and braces?

Reply:
I think we can get rid of undefined, but I think we need null and false, unfortunately. 👎

Reply:
Actually, I got rid of it all with falsy 👍